### ISSUE TITLE
[models] Add __repr__ attribute to GeneralizedRCNNTransform

### ISSUE THREAD

**Issue by frgfm** [pytorch/vision#1786]
## 🚀 Feature
Adding a `__repr__` attribute would allow the user to check resize and normalize parameter easily.

## Motivation
During a training using `FasterRCNN`, I forgot to set the `min_size` and `max_size` argument in the constructor. Unfortunately, my training set only has squared images and with lower dimensions than the default values. When I printed the model for debugging, `GeneralizedRCNNTransform` was shown without any further information, which would have been handy in the first place.

## Pitch
The following commands
```
from torchvision.models.detection.transform import GeneralizedRCNNTransform
t = GeneralizedRCNNTransform(224, 224, [0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
print(t)
```
would yield
```
GeneralizedRCNNTransform(
    Resize(size=(224, 224), interpolation=PIL.Image.BILINEAR)
    Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
)
```

I would have opened a PR with similar implementation as common transforms, but since the `forward` method has other operations than strictly resizing and normalizing, I preferred to get your input first!

**Comment by fmassa** [comment]
Totally agree, could you send a PR implementing this?

> I would have opened a PR with similar implementation as common transforms, but since the forward method has other operations than strictly resizing and normalizing, I preferred to get your input first!

The other things that i does do not have parameters (at least not exposed to the user, like `size_divisible`), so I'd say that a `__repr__` like you propose is enough

**Comment by frgfm** [comment]
Solved by #1834
